ui/src/dashboards/actions/index.js

```diff
@@ -145,6 +164,17 @@ export const putDashboard = dashboard => async dispatch => {
   }
 }
 
+export const putDashboardByID = dashboardID => async (dispatch, getState) => {
+  try {
+    const {dashboardUI: {dashboards}} = getState()
```

Comment:
could you add a comment about why you're using `getState`, having to do with needing to not use a stale dashboard after `updateTempVarValues()`?